build/Dockerfile

```diff
@@ -67,42 +67,36 @@ COPY /docker-entrypoint.sh /entrypoint.sh
 COPY /static $DSTA_HOME
 RUN mkfifo $DSTA_HOME/console \
 	&& chown -R $STEAM_USER:$STEAM_USER $DSTA_HOME \
-	&& mkdir -p `dirname $CONFIG_PATH` \
-	&& chown $STEAM_USER:$STEAM_USER `dirname $CONFIG_PATH`
+	&& mkdir -p `dirname $CLUSTER_PATH` \
+	&& chown $STEAM_USER:$STEAM_USER `dirname $CLUSTER_PATH`
 
 ARG SERVER_PORT=10999
 
 # Set environment variables.
-ENV SERVER_DESCRIPTION="Powered by DST Academy." \
+ENV DESCRIPTION="Powered by DST Academy." \
 	SERVER_PORT="$SERVER_PORT" \
-	OFFLINE_ENABLE=false \
 	MAX_PLAYERS=4 \
-	WHITELIST_SLOTS=0 \
 	PVP_ENABLE=false \
 	GAME_MODE=survival \
-	SERVER_INTENTION=cooperative \
+	INTENTION=cooperative \
 	AUTOSAVER_ENABLE=true \
 	TICK_RATE=15 \
-	CONNECTION_TIMEOUT=5000 \
 	VOTE_KICK_ENABLE=true \
 	PAUSE_WHEN_EMPTY=true \
 	STEAM_AUTHENTICATION_PORT=8766 \
 	STEAM_MASTER_SERVER_PORT=27016 \
-	STEAM_GROUP_ONLY=false \
-	STEAM_GROUP_ADMINS=false \
 	CONSOLE_ENABLE=true \
-	AUTOCOMPILER_ENABLE=false \
-	MODS_ENABLE=true \
 	SHARD_ENABLE=false \
+	SHARD_NAME=shard \
```

Comment:
Is it needed for us, when using Docker, to have an option for the shard-name at all? Can we hard-code it to a default value?